doc/development-context-aware-applications/deep-dive.md

```diff
@@ -1,12 +1,5 @@
 Should you wish to learn much more about this, please check: 
 
    - [Introduction to NGSI API and Orion Context Broker](http://bit.ly/fiware-orion)
-   - [Github repository (reference implementation) ](https://github.com/Fiware/context.Orion)
```

Comment:
Please keep the link to the Github repository as we agreed

Reply:
Thanks, just added